src/zelos/feeds.py

```diff
@@ -0,0 +1,226 @@
+# Copyright (C) 2020 Zeropoint Dynamics
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+
+# You should have received a copy of the GNU Affero General Public
+# License along with this program.  If not, see
+# <http://www.gnu.org/licenses/>.
+# ======================================================================
+
+"""Implementation of feeds.
```

Comment:
Ensure this shows up in documentation. Add a "feed" tutorial.

Reply:
We can add a tutorial once feeds are added to the API